src/App.tsx

```diff
@@ -1,14 +1,110 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { getTodos, USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import cn from 'classnames';
+
+const filterTodos = (todos: Todo[], filter: string) => {
+  switch (filter) {
+    case 'All':
+      return todos;
+
+    case 'Active':
+      return [...todos].filter(el => !el.completed);
+
+    case 'Completed':
+      return [...todos].filter(el => el.completed);
+
+    default:
+      return;
+  }
+};
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [ActiveChangeId, setActiveChangeId] = useState(0);
+  const [editedTodo, setEditedTodo] = useState('');
+  const [newTodo, setNewTodo] = useState('');
+  const [filter, setFilter] = useState('All');
+  const [errors, setErrors] = useState('');
+
+  useEffect(() => {
+    const fetchedTodos = async () => {
+      try {
+        const list = await getTodos();
+
+        setTodos(list);
+      } catch (error) {
+        setErrors('Unable to load todos');
+        setTimeout(() => {
+          setErrors('');
+        }, 3000);
+      }
+    };
+
+    fetchedTodos();
+  }, []);
+
+  function loseFocus(id: number) {
```

Comment:
```suggestion
  const loseFocus = (id: number) => {
```
Try to keep the same style of writing code